pkg/roachprod/cloud/cluster_cloud.go

```diff
@@ -359,6 +359,27 @@ func GrowCluster(l *logger.Logger, c *Cluster, NumNodes int) error {
 	})
 }
 
+// ShrinkCluster removes tail nodes from an existing cluster.
+func ShrinkCluster(l *logger.Logger, c *Cluster, numNodes int) error {
+	providers := c.Clouds()
+	if len(providers) != 1 && providers[0] != gce.ProviderName {
+		return errors.Errorf("cluster %s is not on gce, shrinking a cluster is currently only supported on %s",
```

Comment:
Nit: inconsistency: in one instance the error mentions `gce` directly, in the other it uses `gce.ProviderName`.

Reply:
Yep, this is could use improvement. Done.